lib/completions/endpoints/mistral.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module Completions
+    module Endpoints
+      class Mistral < OpenAi
```

Comment:
Should the `provider_id` be overridden? As-is, this currently returns the one for OpenAi.

Reply:
oh sure ... good catch will fix